source/compiler-core/slang-artifact.h

```diff
@@ -143,6 +143,7 @@ enum class ArtifactPayload : uint8_t
     CUDA,           ///< CUDA source
     Metal,          ///< Metal source
     Slang,          ///< Slang source
+    WGSL,           ///< WGSL source
```

Comment:
This needs to be placed at the end to avoid breaking binary compatibilty I think.